gordo_client/io.py

```diff
@@ -25,24 +21,24 @@ class ResourceGone(Exception):
     Used in our case for calling the server with a revision which is no longer used.
     """
 
-    pass
-
 
 class BadGordoRequest(Exception):
     """
     Represents a general 4xx bad request
     """
 
-    pass
+
+class BadGordoResponse(Exception):
```

Comment:
```suggestion
class BadGordoResponse(ValueError):
```

Reply:
i think now when we check for mode/json response any other type is invalid and it should be derived from Exception